tpr/backoff.go

```diff
@@ -0,0 +1,25 @@
+package tpr
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
+type BackOff struct {
+	retries  int
+	interval time.Duration
+}
+
+func (b *BackOff) Reset() {}
```

Comment:
Is Reset supposed to be an empty func? I don't get it so a comment would be nice.